.nvmrc

```diff
@@ -0,0 +1 @@
+16
```

Comment:
I have taken the freedom to add a [NVM](https://github.com/nvm-sh/nvm) RC file. Let me know if you would like this out